rollbar-spring-webmvc/src/main/java/com/rollbar/spring/webmvc/RollbarWebApplicationInitializer.java

```diff
@@ -0,0 +1,23 @@
+package com.rollbar.spring.webmvc;
+
+import com.rollbar.web.filter.RollbarFilter;
+import com.rollbar.web.listener.RollbarRequestListener;
+import javax.servlet.ServletContext;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.WebApplicationInitializer;
+
+public class RollbarWebApplicationInitializer implements WebApplicationInitializer {
+
+  @Override
+  @Autowired
+  public void onStartup(ServletContext container) {
+
+    // Attach the RollbarRequestListner to attribute HTTP Request data into the Exception object
+    // This will be visible in Rollbar
+    container.addListener(RollbarRequestListener.class);
+    container.addFilter("RollbarFilter", new RollbarFilter())
```

Comment:
We don't need to declare the filter. For this integration I think that filter doesn't provide anything as with Spring we will track the unhandled exceptions by using the `HandlerExceptionResolver` implementation.

Reply:
done